app/templates/shapeshift/confirm.pug

```diff
@@ -10,7 +10,7 @@ form.bc-form(
       span.pr-30.flex-row.flex-center.no-padding-mobile(translate="Step 2 of 2")
     .ph-30.pt-30.ph-10-mobile.pv-20-mobile
       .f-13.mv-20
-        span(translate=".CONFIRM_DEPOSIT_AMOUNT" translate-values="{deposit: total, widthdrawal: quote.withdrawalAmount, input: quote.fromCurrency, output: quote.toCurrency, from: from.label, to: to.label}")
+        span(translate=".CONFIRM_DEPOSIT_AMOUNT" translate-values="{deposit: quote.depositAmount, withdrawal: quote.withdrawalAmount, input: quote.fromCurrency, output: quote.toCurrency, from: from.label, to: to.label}")
       .flex-end.mb-5
```

Comment:
Funny misspelling, thanks for catching that.

Do we want to display the depositAmount or the total?